# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/painting/RetainedDisplayListBuilder.h
# Commit: 1d600b2ed81f
# Full Hash: 1d600b2ed81f163645a033b2e23ee2ac325ef3c1
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-02-02 09:43:12
# Regressor Bug: 1860328
# File Overlap Count: 2
# Description:
#   Bug 1860328 - Fix a bug where caret misses when dragging and dropping r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192377
# ==============================================================================

diff -r 854c462a083b -r 1d600b2ed81f layout/painting/RetainedDisplayListBuilder.h
--- a/layout/painting/RetainedDisplayListBuilder.h	Thu Feb 01 18:01:46 2024 +0000
+++ b/layout/painting/RetainedDisplayListBuilder.h	Thu Feb 01 18:09:39 2024 +0000
@@ -272,7 +272,6 @@
 
   nsDisplayListBuilder mBuilder;
   RetainedDisplayList mList;
-  WeakFrame mPreviousCaret;
   RetainedDisplayListMetrics mMetrics;
   RetainedDisplayListData mData;
 };